app/helpers/application_helper/button/host_feature_button.rb

```diff
@@ -2,7 +2,6 @@ class ApplicationHelper::Button::HostFeatureButton < ApplicationHelper::Button::
   needs :@record
 
   def visible?
```

Comment:
I have removed host_feature_button and host_feature_button_with_disable as they aren't needed anymore.